collections/hub/HA/golden_path/fx/feature_tests/happy_path/fx_tests.json

```diff
@@ -916,6 +916,7 @@
           },
           "url": "{$inputs.HOST_CENTRAL_LEDGER}",
           "scriptingEngine": "javascript",
+          "retries": 5,
```

Comment:
I think if previous request gets a callback, there shouldn't be need for retrying to get the updated positions. If this test is failing intermittently due to position check, I think we need to check why its happening instead of adding retries here.

Reply:
removed for now